__tests__/app.test.js

```diff
@@ -41,20 +41,20 @@ describe("GET /api/topics", () => {
   test("400: returns bad request when nothing is passed through", () => {
```

Comment:
The test is written as a `400` but you're expecting `404`. You've written that nothing is passed through - what does that mean exactly? It looks to me like the test is checking what happens when an invalid endpoint is called